client/lib/plans/plans-list.js

```diff
@@ -932,52 +932,22 @@ export const PLANS_LIST = {
 	},
 
 	[ constants.PLAN_JETPACK_FREE ]: {
-		term: constants.TERM_ANNUALLY,
```

Comment:
Removing the term prevents us from creating a new entry for the monthly term, which seemed confusing given the plan is free.